modules/pages/client/components/FaqFoundation.component.js

```diff
@@ -10,19 +10,10 @@ export default function FaqFoundation() {
       <div className="faq-question" id="what-is-your-legal-status">
         <h3>{t("What's your legal status?")}</h3>
         <Trans t={t} ns="pages">
-          As of May 2022{' '}
-          <a href="https://find-and-update.company-information.service.gov.uk/company/09489825/filing-history">
-            Trustroots Foundation is dissolved
-          </a>
-          . What does it mean for the community? We don’t know exactly yet!
-        </Trans>
-        <br />
-        <br />
-        <Trans t={t} ns="pages">
-          Prior to that, the website was owned and operated by{' '}
-          <a href="/foundation">Trustroots Foundation</a>, a non-profit Limited
-          by Guarantee (LBG) under section 60 exemption, registered in the
-          United Kingdom in March 2015.
+          The website is owned and operated by{' '}
+          <a href="/foundation">Trustroots Foundation</a>, a non-profit
+          Limited by Guarantee (LBG) under section 60 exemption, registered in
+          the United Kingdom since March 2015. However, the Trustroots Foundation has been inactive since the 24th of May 2022. This is why we are considering to either establish a new foundation, migrating to a p2p decentralized solution or reactivating the existing foundation one to resume its operations. Meanwhile, a small core team, consisting of members who have been involved with Trustroots since its early days, is currently managing the <a href="https://team.trustroots.org/Server.html">server and user database</a>.
```

Comment:
In `reactivating the existing foundation one to resume`, remove `one`

Also it would be more correct to replace `existing` with `former` or `dissolved`